js/utils/check-string-length.js

```diff
@@ -1,3 +1,4 @@
 function checkStringLength (string, length) {
```

Comment:
```suggestion
const checkStringLength (string, length) => {
```

Нам же нужно придерживаться одного стиля объявления функций, который мы выбрали. У нас большинство это стрелочные функции, значит все нужно так реализовать и реализовывать.
https://up.htmlacademy.ru/javascript/23/criteries#d5